CHANGES.rst

```diff
@@ -23,6 +23,8 @@ Unreleased
 -   Roll back a change to the order that URL matching was done. The
     URL is again matched after the session is loaded, so the session is
     available in custom URL converters. :issue:`4053`
+-   Use previously unused `escape` import statement in the quickstart
+    page of the docs. :issue: `4054`
```

Comment:
I don't think documentation updates belong in the changelog. They are irrelevant to users of a library, who are the main audience for the changelog.

Reply:
> I don't think documentation updates belong in the changelog. They are irrelevant to users of a library, who are the main audience for the changelog.

Should I make a new pull request?

Reply:
no no, just remove the commit and force-push (or revert it, we can squash before merging)